File: c++/eye_stream.cpp
int eye_stream_main()

{

str url = "rtsp://192.168.71.50:8554/live/eyes"
cv::VideoCapture capture(url);

if (!capture->isOpened()) {
    //Error
}

cv::namedWindow("eyes", CV_WINDOW_AUTOSIZE);

cv::Mat frame;

while(m_enable) {
    if (!capture->read(frame)) {
        //Error
    }
    cv::imshow("eyes", frame);

    cv::waitKey(30);
}

}

File: python3/eye_stream.py

import cv2


cap = cv2.VideoCapture("rtsp://192.168.71.50:8554/live/eyes")

while True:
    if cap.isOpened():
        ret, frame = cap.read()

        frame = cv2.resize(frame, (480, 640))
        cv2.imshow('rtsp eye stream',frame)
        cv2.waitKey(1)

        k = cv2.waitKey(5) & 0xFF
        if k == ord('e'):
            cv2.destroyWindow('rtsp eye stream')
            break

    else:
        pass
        #print("No Stream opened")
